Decred Wallet/Features/Security/PinInputView.swift

```diff
@@ -8,98 +8,96 @@
 
 import UIKit
 
-class PinInputView: UIView {
```

Comment:
Delete this file if it's no longer needed.

Reply:
resolved in a0eb56d